airflow/providers/amazon/aws/hooks/base_aws.py

```diff
@@ -49,9 +50,19 @@ class AwsBaseHook(BaseHook):
     :type verify: str or bool
     """
 
-    def __init__(self, aws_conn_id="aws_default", verify=None):
+    def __init__(
+            self,
+            aws_conn_id="aws_default",
+            verify=None,
+            region_name=None,
+            client_type=None,
+            resource_type=None
```

Comment:
Please update the documentation accordingly.

Reply:
And also update the docs of the hooks which are based on this hook.

Reply:
Thanks, I forgot to do that step :p Am busy with it now.

Reply:
Good catch @feluelle 😄

Reply:
@feluelle Thanks for the feedback, please take another look. I've updated the docstrings on all the hooks accordingly. There's only a "how to" document for the DataSync operator, no other separate docs for AWS - at least that I could see. Let me know if there's anywhere else you can think of :)